docs/reference/index-modules/allocation/data_tier_allocation.asciidoc

```diff
@@ -29,16 +29,22 @@ mounted indices>> exclusively.
     Assign the index to a node whose `node.roles` configuration has at
     least one of to the comma-separated values.
 
+    deprecated::[7.13, Filtering based on a single tier is deprecated, use <<tier-preference-allocation-filter,_tier_preference>> instead]
```

Comment:
I find the "single tier" part here slightly confusing in that the setting could accept multiple tiers.

```suggestion
    deprecated::[7.13, Filtering based on a `include._tier`, `require._tier` and `exclude._tier` is deprecated, use <<tier-preference-allocation-filter,_tier_preference>> instead]
```